code/src/App.js

```diff
@@ -1,9 +1,77 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import { API_URL, LIKES_URL } from 'utils/urls'
+import Form from 'Components/Form'
+import Messages from 'Components/Messages'
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([])
+  const [messageNew, setMessageNew] = useState('')
+
+  useEffect(() => {
+    fetchMessageList()
+  }, [])
+
+  const fetchMessageList = () => {
+    fetch(API_URL)
+      .then(result => result.json())
+      .then(messages => setMessageList(messages))
+      .catch(error => console.error(error))
+  }
+
+  const handleMessageNewChange = (event) => {
+    setMessageNew(event.target.value)
+  }
+
+  const refreshPage = () => {
+    window.location.reload()
```

Comment:
It is an anti-pattern to reload single page application. You need to rethink the state/props structure to make your app re-render